apps/journeys-admin/src/components/AiChat/MessageList/ToolInvocationPart/agent/GenerateImageTool/GenerateImageTool.tsx

```
@@ -0,0 +1,44 @@
+      return (
+        <Stack gap={2} direction="row">
+          {part.toolInvocation.result.map((image) => (
+            <Image
+              src={image.src}
+              alt="Generated image"
+              width={256}
+              height={256}
+              style={{
+                borderRadius: 5
+              }}
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)